sympy/stats/stochastic_process_types.py

```diff
@@ -1465,7 +1465,7 @@ def canonical_form(self) -> tTuple[tList[Basic], ImmutableMatrix]:
         ==========
 
         .. [1] https://onlinelibrary.wiley.com/doi/pdf/10.1002/9780470316887.app1
-        .. [2] http://www.columbia.edu/~ww2040/6711F12/lect1023big.pdf
+        .. [2] https://web.archive.org/web/20220207000124/http://www.columbia.edu/~ww2040/6711F12/lect1023big.pdf
```

Comment:
I'm not sure why web archive is needed for this though. The HTTP link works, although the website may not support HTTPS link though.

Reply:
Ok, I am reverting this web archive link. Apparently I didn't realize that my university Wi-Fi was specifically redirecting the HTTP link of this domain to HTTPS. For others web archived links I was able to open the HTTP and HTTPS links but not for this one and I didn't realize that fact.